src/macros.rs

```diff
@@ -105,6 +105,22 @@ macro_rules! register_counter {
 }
 
 #[macro_export]
+macro_rules! register_counter_vec {
+    ( $ OPTS : expr , $ LABELS_NAMES : expr ) => {
+        {
+            let counter_vec = $crate::CounterVec::new($OPTS, $LABELS_NAMES).unwrap();
+            $crate::register(Box::new(counter_vec.clone())).map(|_| counter_vec)
+        }
+    };
+
+    ( $ NAME : expr , $ HELP : expr , $ LABELS : expr , $ LABELS_NAMES : expr ) => {
```

Comment:
Just pass `labels!{}` for omitting const_labels.

I think it is pretty confusing if we support 
`( $ NAME : expr , $ HELP : expr , $ LABELS : expr , $ LABELS_NAMES : expr )` and
`( $ NAME : expr , $ HELP : expr , $ LABELS_NAMES : expr )` as the same time.


It is too easy for someone to write something like `register_counter_vec!("name", "help", labels!{"a" => "b",})`